inst/stan/m_exp_covar.stan

```diff
@@ -93,10 +93,10 @@ generated quantities {
   
   // Restricted mean survival time
   for (t in 1:tg) {
-    S_0N[t] = 1-exponential_cdf(t_grid[t], lambda_0N);
-    S_1N[t] = 1-exponential_cdf(t_grid[t], lambda_1N);
-    S_0T[t] = 1-exponential_cdf(t_grid[t], lambda_0T);
-    S_1T[t] = 1-exponential_cdf(t_grid[t], lambda_1T);
+    S_0N[t] = 1-exponential_cdf(t_grid[t] | lambda_0N);
```

Comment:
The [functions reference](https://mc-stan.org/docs/functions-reference/exponential-distribution.html) appears to still state `exponential_cdf(reals y, reals beta)`, but it looks inconsistent and might be a documentation error(?).

Reply:
Very good catch, that is a doc error! I'll make a note to get that updated. Thanks!